SPECS/rubygem-rexml/rubygem-rexml.spec

```diff
@@ -3,13 +3,14 @@
 Summary:        REXML is an XML toolkit for Ruby
 Name:           rubygem-%{gem_name}
 Version:        3.3.4
```

Comment:
The [CVE advisory ](https://github.com/ruby/rexml/security/advisories/GHSA-2rxp-v6pw-ch6m) suggests that version 3.3.9 is already fixed. This is only a patch version bump, so I'd suggest doing that instead of a single patch.

Reply:
Thanks for the call out @PawelWMS  ! closing this in favor of https://github.com/microsoft/azurelinux/pull/13851